src/behaviors/Behavior.js

```diff
@@ -721,13 +717,13 @@ var Behavior = Base.extend('Behavior', {
      * @memberOf Behavior.prototype
      * @desc update the data at point x, y with value
      * @return The data.
-     * @param {number} x - x coordinate
-     * @param {number} y - y coordinate
+     * @param {number} c - Grid column coordinate.
+     * @param {number} r - Grid row coordinate.
      * @param {Object} value - the value to use
      */
-    setValue: function(x, y, value) {
-        var column = this.getActiveColumn(x);
-        return column && column.setValue(y, value);
+    setValue: function(c, r, value) {
```

Comment:
Let's gradually change `x` (data column index) to `c` (grid column index) as appropriate and/or `y` (data row index) to `r` (grid row index) as appropriate.

This will really help make the code more comprehensible. This is just a start...

Importnat note: These coordinate changes do _not_ necessarily go in tandem as some important methods (such as `getCell`) currently use `x` and `r`.